include/classes/DAO/DAO.class.php

```diff
@@ -63,22 +50,29 @@ function DAO()
 	function execute($sql)
 	{
 		$sql = trim($sql);
-		$result = mysql_query($sql, $this->db) or die($sql . "<br />". mysql_error());
+		$result = mysqli_query($this->db, $sql) or die($sql . "<br />". mysqli_error());
 
 		// Deal with "select" statement: return false if no row is returned, otherwise, return an array
 		if ($result !== true && $result !== false) {
 			$rows = false;
 			
-			while ($row = mysql_fetch_assoc($result)){
+			while ($row = $result->fetch_assoc()){
 				if (!$rows) $rows = array();
 				
 			    $rows[] = $row;
 			}
-			mysql_free_result($result);
+			mysqli_free_result($result);
 			return $rows;
 		}
 		return true;
 	}
+	
+	function addSlash($code)
```

Comment:
"addSlash" -> "addSlashes". Add "es" to the function name for consistence.

It's a good practice to create this `addSlashes` as a DAO base method since this method will only be called in DAO classes.

Reply:
will it be good if i create dbFunction.class that do anything needs a db connection to the non DAO classes ? so i can use addslashes with the index.php file